node_modules/oae-activity/lib/internal/dao.js

```diff
@@ -77,23 +77,26 @@ var getActivities = module.exports.getActivities = function(activityStreamId, st
  * @param  {Object}     callback.activitiesPerStream    The list of activities keyed by their activity stream id
  */
 var getActivitiesFromStreams = module.exports.getActivitiesFromStreams = function(activityStreamIds, start, callback) {
-    var query = 'SELECT * FROM "ActivityStreams" WHERE "activityStreamId" IN (?)';
-    var parameters = [ activityStreamIds ];
+    var placeholders = _.map(activityStreamIds, function() {
+        return '?';
+    });
+    var query = util.format('SELECT * FROM "ActivityStreams" WHERE "activityStreamId" IN (%s)', placeholders.join(','));
+    var parameters = activityStreamIds.slice();
     if (start) {
         query += ' AND "activityId" > ?';
         parameters.push(start + ':');
     }
 
-    Cassandra.runQuery(query, parameters, function(err, rows) {
+    Cassandra.runQuery(query, parameters, {'cache': false}, function(err, rows) {
```

Comment:
Is it possible that, if we use `runInQuery` query for this kind of case and never cache IN queries, that we would not have to add an additional parameter to `runQuery`?